### ISSUE TITLE
NTR: Arthrocentesis: synovial fluid crystals (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8658]
New term request Arthrocentesis: synovial fluid crystals (athrey_140317132002): Uric acid; calcium pyrophosphate in joint fluid  UMLS: 0428078

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000248
- label: Synovial fluid crystals
- parents: [Abnormal joint morphology (HP:0001367)](https://hpo.jax.org/app/browse/term/HP:0001367)
- pmid: PMID:15769916
- def: The presence of crystals (microscopic solids that formdistinct lattices, with definable fixed angles, faces, walls) in the joint (synovial) fluid.
- comment: Polarized light microscope is the standard method for synovial fluid analysis in the search for crystals. Identification of monosodium urate and calcium pyrophosphate crystals in synovial fluid samples obtained from inflamed joints allows the precise diagnosis of gout and calcium pyrophosphate dehydrate crystal related arthropathy
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8658
- synonyms:

**Comment by MickeySegal** [comment]
In def need space inside "formdistinct"

**Comment by pnrobinson** [comment]
thanks for pointing out the error, reopening.

**Comment by pnrobinson** [comment]
fixed!
